modules/cluster-logging-deploying-about.adoc

```diff
@@ -155,10 +155,11 @@ spec:
       nodeCount: 3
       resources:
         limits:
-          memory: 32Gi
+          cpu: 200m
+          memory: 16Gi
         requests:
-          cpu: 3
```

Comment:
As the plan, https://access.redhat.com/support/policy/updates/openshift_operators.  Elasticsearch-opertaor in 4.16,4.17 is for trace only.   I think we needn't remove all section as elasticsearch as default storge.